aviary/subsystems/propulsion/propeller_map.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np
+from enum import Enum
+import warnings
+import openmdao.api as om
+from openmdao.utils.units import convert_units
+
+from aviary.variable_info.options import get_option_defaults
+from aviary.variable_info.variables import Aircraft, Dynamic, Settings
+from aviary.utils.aviary_values import AviaryValues, NamedValues, get_keys, get_items
+from aviary.utils.csv_data_file import read_data_file
+from aviary.utils.functions import get_path
+import pdb
+
+class PropModelVariables(Enum):
+    '''
+    Define constants that map to supported variable names in a propeller model.
+    '''
+    MACH = 'Mach_Number'
+    CP = 'CP'  # power coefficient
+    CT = 'CT'  # thrust coefficient
+    J = 'J'  # advanced ratio
+
+
+default_units = {
+    PropModelVariables.MACH: 'unitless',
+    PropModelVariables.CP: 'unitless',
+    PropModelVariables.CT: 'unitless',
+    PropModelVariables.J: 'unitless',
+}
+
+MACH = PropModelVariables.MACH
+CP = PropModelVariables.CP
+CT = PropModelVariables.CT
+J = PropModelVariables.J
```

Comment:
If you switch the order of these, you can use the shortened variable names in the default_units dict
```suggestion
MACH = PropModelVariables.MACH
CP = PropModelVariables.CP
CT = PropModelVariables.CT
J = PropModelVariables.J

default_units = {
    MACH: 'unitless',
    CP: 'unitless',
    CT: 'unitless',
    J: 'unitless',
}
```

Reply:
agree.